docs/guides/force-organizations.mdx

```diff
@@ -148,134 +200,116 @@ export function SyncActiveOrganization() {
 }
 ```
-    </Tab>
-</Tabs>
-
-### Detect an active organization
 
-A session will always include an organizationId via the [`AuthenticationObject`](/docs/references/nextjs/authentication-object#example-with-active-organization), we can use this in order to detect if a user has an "active organization".
+### Limit access to users with active organizations only
 
-<Tabs type="router" items={["App Router Server Component", "App Router Client Component"]}>
-    <Tab>
-```tsx
-import { auth } from '@clerk/nextjs/server'
+Now that you have hidden personal accounts from the UI and can detect and set an active organization, you can limit access to users with active organizations only.
 
-export default function Layout() {
-  const { orgId } = auth()
-  const hasActiveOrg = orgId !== null
-  ...
-}
-```
-    </Tab>
-
-    <Tab>
-```tsx
-"use client"
-import { useAuth } from '@clerk/nextjs'
+It's possible for a user to be logged in, but not have an active organization. This can happen in two cases:
+  - The user created an account and hasn't created an organization yet
+  - The user joined or created multiple organizations, and left or deleted the active organization
 
-export default function Layout() {
-  const { orgId } = useAuth()
-  const hasActiveOrg = orgId !== null
-  ...
-}
-```
-    </Tab>
-</Tabs>
+Based on your use case, you can decide to limit users either in the entire app or a specific part of it. For example, a B2B application might need to limit access to only users with an active organization, whereas a B2B2C application might limit only the `/dashboard` path to users with an active organization.
 
+#### Limit the entire application to only users with active organizations
 
-### Limit access for users with active organizations only
+In Next.js App Router applications, you can use a [layout](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#layouts) to limit the entire application to only users with active organizations.
 
-<Callout type="info">
-  **How users can end up in this state ?**
-  - User just created an account, not by accepting an invitation, and hasn’t created an organization yet
-  - User has joined or created multiple organizations and leaves or deletes the active organization. 
-</Callout>
+In the example below, a component called `RequiredActiveOrgLayout` is created. This component will be used as a layout for all pages that require an active organization. If the user has an active organization, the children will be rendered. If the user does not have an active organization, the organization selection page will be rendered.
 
-Based on your usecase you can either decide to limit users in the entire app or a specific part of it. For example a B2B application might need to limit users with active organization for the entire app, whereas a B2B2C application would only limit the `/dashboard` path to users with active organization.
+```tsx filename="app/(with-active-organization)/layout.tsx"
```

Comment:
As this section is for 'the whole application' this should probably be modified to be in the root layout. That or something more like "app/(application)/layout.tsx"

Also see comment below.